cmd/ferretdb/ping.go

```diff
@@ -95,46 +95,43 @@ func ping() {
 			RawQuery: values.Encode(),
 		}
 
-		urls = append(urls, u.String())
+		urls = append(urls, u)
 	}
 
 	if cli.Listen.Unix != "" {
 		l.Debugf("--listen-unix flag is set. Ping to %s will be performed.", cli.Listen.Unix)
 
-		urls = append(urls, "mongodb://"+url.PathEscape(cli.Listen.Unix))
+		urls = append(urls, &url.URL{Scheme: "mongodb", Path: url.PathEscape(cli.Listen.Unix)})
 	}
 
 	if len(urls) == 0 {
-		l.Info("Neither --listen-addr nor --listen-unix flags were specified - skipping ping.")
+		l.Info("None of --listen-addr, --listen-unix nor --listen-tls flags were specified - skipping ping.")
 		return
 	}
 
 	for _, u := range urls {
-		l.Debugf("Pinging %s...", u)
+		redactedURL := u.Redacted()
+		l.Debugf("Pinging %s...", redactedURL)
 
 		ctx, cancel := context.WithTimeout(context.Background(), cli.Setup.Timeout)
 		defer cancel()
 
-		client, err := mongo.Connect(ctx, options.Client().ApplyURI(u))
+		client, err := mongo.Connect(ctx, options.Client().ApplyURI(u.String()))
 		if err != nil {
-			l.Fatal("Connection failed", zap.Error(err))
+			logger.Fatal("Connection failed.", zap.Error(err))
```

Comment:
Using `logger.Fatal(...)`  uses zap.Field appropriately like this
```
FATAL   ferretdb/main.go:375    Connection failed.      {"error": "some error"}
```
Sugared `l.Fatal(...)` is expected for Fatalf usage, so error displayed string concatenated and isn't friendly. 
```
FATAL   ferretdb/main.go:375    Connection failed.{error 26 0  some error}
```